examples/fluids/blasius.yaml

```diff
@@ -0,0 +1,36 @@
+problem: 'blasius'
+
+#stab: 'supg'
+Ctau_t: 1
+#Ctau_v: 36,60,128 is what PHASTA has for p=1,2, 3
+Ctau_v: 36
+Ctau_C: 0.125
+Ctau_M: 1.0
+Ctau_E: 0.125
+q_extra: 0
+implicit: true
+ts_max_snes_failures: 0
+ts_adapt_type: 'none'
+ts_type: 'beuler'
+ts_dt: 5.e-8
```

Comment:
I tried running it with an advective CFL below 1 (dt = 2e-6, CFL dt = 2.6e-6 = 4.2e-3(m)/40 (streamwise elements) / 40(m/s) ), but it's quite unstable. It actually rejected a few iterations and dropped the timestep all the way down to 1.2e-7, but was still having issues even at that timestep. 

Looks like there's a pressure wave that needs to be resolved right at the beginning of the simulation. I'm trying it was a less aggressive initial timestep now (5e-7).

Reply:
Even 2e-7 forces itself to reduce the timestep. I'm honestly not sure what's happening here, as I had it working fine a few days ago.

Reply:
Did you increase Mach number while keeping Reynolds constant, or are you increasing Reynolds while speeding up the flow? Can you push/comment with what you're trying and I'll run the same?

Reply:
The only thing I'm changing here is the timestep. I've gotten 1e-7 to run stably for a while (around 1900 timesteps), but anything above that seems to diverge before it hits 200 timesteps (if not sooner).

I'm not sure why the initial pressure wave is happening either.

Reply:
Is this missing any BC stuff from @KennethEJansen's branch?
![image](https://user-images.githubusercontent.com/3303/165886729-d3f03c92-890d-4248-97aa-3c1ffb27d213.png)

I think the time step here is too small because tau_m will become way too small. My understanding is that the chosen tau is meant to be run around advective CFL 1. That said, I'm not getting stable solutions with bigger steps either. That situation is better in `ken/densityInflow`, where energy and velocity seem stable and I get this density profile before getting a zero pivot in the subdomain solver. (The linear iteration counts are always high, in the 200 range, though Newton is fast) The default time step is the same in that branch and seems tiny for this problem (streamline CFL for a wave of speed 2km/s.)

```console
$ mpiexec -n 4 ./navierstokes -options_file blasius.yaml -ts_adapt_monitor -ts_dt 2e-6 -snes_monitor -ksp_converged_reason
```
![image](https://user-images.githubusercontent.com/3303/165886948-ff3b0c61-7b18-43d5-862a-9ed06420d8b8.png)

Reply:
Given that the `ken/densityInflow` seems to get good results, should we go ahead and merge this in, then merge `ken/densityInflow` soon after? Or were you still wanting to figure something out with the weak-temperature inflow?

Reply:
Above is Hauke and Hughes (from Franca and Frey) tau which is not what I coded.

Reply:
Hmm, that image is from Whiting-Jansen-Dey 2003 (eq 41-44). My point was just that tau_m can get small if the time step gets too small. Or is that not an issue due to grading in y?